modules/client/examples/05-encoders-decoders/10-upgrade-to.ts

```diff
@@ -1,6 +1,6 @@
 /* MARKDOWN
 ---
-title: Upgrade To
+title: Upgrade DAO
```

Comment:
This is not specific to the DAO contract. Any UUPS upgradeable contract could be targeted here. I suggest we leave it like:

```suggestion
title: Upgrade contract
```